### ISSUE TITLE
[bitnami/postgresql] Default pg_hba.conf does not accept connections over IPv6

### ISSUE THREAD

**Issue by stv0g** [bitnami/charts#2308]
**Which chart**: postgresql-8.7.3

**Describe the bug**
Default pg_hba.conf does not accept connections over IPv6.

**To Reproduce**

Deploy a postgresql release with default values.

**Expected behavior**:

I dont see a reason why we only accept connection for IPv4 in 2020.
The `pg_hba.conf` should include the following line:

```
host     all     all     ::/0     md5
```

**Version of Helm and Kubernetes**: 1.18.0

- Output of `helm version`: 

```
version.BuildInfo{Version:"v3.1.1", GitCommit:"afe70585407b420d0097d07b21c47dc511525ac8", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:14:22Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:50:46Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by marcosbc** [comment]
Hi @stv0g, we do enable IPv6 but it seems like we are only allowing remote connections for IPv4, so your request makes sense. We're working on performing the changes and will keep you posted with the progress.

**Comment by marcosbc** [comment]
@stv0g We just released a new version of the container image (`11.7.0-debian-10-r71`) and chart (`8.7.4`) with a fix to this. Thank you for the bug report! If you continue to find issues please let us know.
